forms-flow-api/src/formsflow_api/services/process.py

```diff
@@ -481,3 +674,52 @@ def get_process_by_id(cls, process_id):
         if process:
             return processSchema.dump(process)
         raise BusinessException(BusinessErrorCode.PROCESS_ID_NOT_FOUND)
+
+    @classmethod
+    def migrate(cls, request):
+        """Migrate by process key."""
+        current_app.logger.debug("Migrate process started..")
```

Comment:
Migration needs to happen only for all the other mappers with same process_key and keep the same for the one which initiates it. For e.g, if Workflow1 is used by Form1, Form2 and Form3. Then when the user saves Form2 and decides to keep the workflow for that form, then Form1 and Form3 needs new workflow (cloned)

Reply:
![image](https://github.com/user-attachments/assets/126c778f-e9c1-4d8e-aa63-9d66badbe989)
yes picking the mappers excluding the mapperId provided in the payload & adding the new workflow for other form mappers